File: CMOWebApi.Models/AdminModel/MasterModel/DepartmentAuthoritySignatoryModel.cs
using System;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
    public  class DepartmentAuthoritySignatoryModel
    {
        public long Id { get; set; }
        public Nullable<long> Code { get; set; }
        public Nullable<int> DepartmentCode { get; set; }
        public string Name { get; set; }
        public string NameHindi { get; set; }
        public string Designation { get; set; }
        public string DesignationHindi { get; set; }
        public Nullable<bool> IsActive { get; set; } = true;
        public Nullable<bool> IsDeleted { get; set; } = false;
    }

    public class DepartmentAuthoritySignatoryViewModel
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Designation { get; set; }
        public string DepartmentTitle { get; set; }
		public Nullable<bool> IsActive { get; set; }
		public Nullable<bool> IsDeleted { get; set; }
	}
}

File: ANGULAR/ANGULAR/src/app/Shared/Model/Master/departmentauthoritysignatory.model.ts
export class DepartmentAuthoritySignatoryModel {
    Id: number;
    Code: number;
    Name: string;
    NameHindi: string;
    Designation: string;
    DesignationHindi: string;
    DepartmentCode: string|number;
}

export class DepartmentAuthoritySignatoryViewModel {
    Id: number;
    Name: string;
    Designation: string;
    DepartmentTitle: string;
    IsActive:number;
    IsDeleted:number;
}
